### PR TITLE
Tidy some test imports

### PR DESCRIPTION
**Pull request by oalders**
### Summary
Tidying up test imports using https://metacpan.org/pod/perlimports

### Motivation
I've documented the motivation for these changes at https://metacpan.org/pod/App::perlimports

### References
#1747 

I'd be interested in knowing if you find these changes useful, objectionable, a waste of time or some combination of the above.

### CODE REVIEW

File: t/mojo/json_xs.t [hunk 1]
```diff
@@ -16,8 +16,8 @@ sub TO_JSON { shift->something }
 
 package main;
 
-use Mojo::ByteStream;
-use Mojo::Util qw(decode encode);
+use Mojo::ByteStream ();
```
Comment (kraih) [inline t/mojo/json_xs.t:19]: This one is wrong though, there's no default export.

File: t/mojo/roles.t [hunk 2]
```diff
@@ -47,10 +47,10 @@ sub hello {'hello mojo!'}
 
 package main;
 
-use Mojo::ByteStream;
-use Mojo::Collection;
+use Mojo::ByteStream ();
+use Mojo::Collection ();
```
Comment (kraih) [inline t/mojo/roles.t:51]: Also wrong, no default export.

Comment (oalders) [reply t/mojo/roles.t:51]: The idea here is that even though the bare parens actually don't do anything, they serve to document that this statement will not be the source of any imported symbols. For someone already familiar with those classes it's obvious, but for someone else maybe not so much.

Having said that, I can force-push some changes with these removed. What I'm trying to do is get an idea of what people prefer when it comes to imports and this is helpful feedback.

Comment (jberger) [reply t/mojo/roles.t]: I don't think it matters in this case but doesn't calling use with bare parens prevent calling import? That would indeed change behavior in some cases

Comment (oalders) [reply t/mojo/roles.t]: Right. The bare parens can actually change the behaviour, since `import()` won't get called at all in that case.

**Comment by oalders** [comment]
Thanks for the feedback! I'm going to close this now.

**Pull request closed without merging**